data/helper-data.js

```diff
@@ -0,0 +1,252 @@
+const ingredientsData = [
+  {
+    "id": 11,
+    "name": "salt",
+    "estimatedCostInCents": 343
+  },
+  {
+    "id": 22,
+    "name": "sugar",
+    "estimatedCostInCents": 293
+  },
+  {
+    "id": 33,
+    "name": "butter",
+    "estimatedCostInCents": 615
+  },
+  {
+    "id": 44,
+    "name": "green beans",
+    "estimatedCostInCents": 535
+  },
+  {
+    "id": 55,
+    "name": "garlic",
+    "estimatedCostInCents": 251
+  },
+  {
+    "id": 66,
+    "name": "milk",
+    "estimatedCostInCents": 771
+  },
+  {
+    "id": 77,
+    "name": "cheddar cheese",
+    "estimatedCostInCents": 603
+  },
+  {
+    "id": 88,
+    "name": "olives",
+    "estimatedCostInCents": 1003
+  },
+  {
+    "id": 99,
+    "name": "italian herbs",
+    "estimatedCostInCents": 309
+  },
+  {
+    "id": 1010,
+    "name": "cumin",
+    "estimatedCostInCents": 972
+  },
+];
+
+const recipeData = [ 
+  {
+    "id": 111,
+    "image": "https://imageForRecipeOne",
+    "ingredients": [
+      {
+        "id": 11,
+        "quantity": {
+          "amount": 1,
+          "unit": "cup"
+        }
+      },
+      {
+        "id": 88,
+        "quantity": {
+          "amount": 3,
+          "unit": "cups"
+        }
+      },
+      {
+        "id": 22,
+        "quantity": {
+          "amount": 1,
+          "unit": "tbsp"
+        }
+      },
+      {
+        "id": 1011,
+        "quantity": {
+          "amount": 8,
+          "unit": ""
+        }
+      }
```

Comment:
We might run into issues when a recipe requires an ingredient that is not in the ingredients array. We'll likely need to introduce error handling so that the unknown ingredient is pushed to the ingredients array or something like that.